data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,308 @@ mission "Coalition License Patch"
+	on complete
+		conversation
+			`Though on <origin> you weren't greeted by him, the same Arach from the other trips now contacts you as you prepare to land on <planet>.`
+			`	"Captain <last>! Forgive me for my absence in our latest deal, you must. Sort out some business here, I had to," he says.`
+			`	He directs you to land on a separate area of the planet, and halfway there little by little a Heliarch fleet starts tailing your ship. You spend a few minutes hovering over the area until they finally give you permission to land. Some of the ships land along with you while others remain in flight, as you come to a massive warehouse complex on a large valley. The facility is walled with some odd kind of barbed wire on top of the walls and fences, and several armed Heliarch agents are patrolling the area.`
+			`	As you leave your ship, the Arach is waiting for you alongside some Heliarch officers. They have the cargo removed from your ship without saying much.`
+			`	Right before the Heliarch ships prepare to escort you back to the standard docking area, he says, "In the spaceport, meet you I will."`
+
+
+
+mission "Coalition Yottrite 4"
+	name "Yottrite Research"
+	description "Lugglop, one of the Arachi that oversees the mining operations in Bloptab, asked that you pick up some friends of his on <destination> so that they may help with the Yottrite research."
+	to offer
+		has "Coalition Yottrite 3: done"
+	source "Delve of Bloptab"
+	destination "Shadowed Valley"
+	on offer
+		conversation
+			`You wait around the spaceport for some minutes before finally spotting the Arach. He invites you to his office.`
+			`	"Responsible for overseeing deliveries of such exotic materials to the warehouses here, I also am. For the inconvenience and bureaucracy, I apologize," he says. "Forget to properly introduce myself, I often do. Lugglop, I am called."`
+			choice
+				`	"Nice to meet you, I suppose."`
+				`	"Do you need more samples? Is that why you called me here?"`
+			`	"Harder to keep track of them, it becomes, the more samples there are," he says. "Notified by the Heliarchs, I was. No more samples, for now, I can request. Risk letting them be stolen, they cannot." He then shows you a map of the Coalition, pointing to a system in Saryd space. "Use more delicate procedures and research, to study the samples we must. For that, friends of mine on <destination>, will be waiting," he says. "Bring them here, would you, Captain?"`
+			choice
+				`	"Of course, they'll be here soon."`
+					accept
+				`	"Sorry, I'm not one to take passenger jobs."`
+					decline
+	on complete
+		conversation
+			`When you land on <planet>, Lugglop contacts you and informs you that he has already told his friends about you. He says they will be waiting in the spaceport.`
+
+
+
+mission "Coalition Yottrite 5"
+	name "Saryd Specialists"
+	description "Transport the twins Nasilor and Alituri to <destination>, where they will assist in the Yottrite research."
+	passengers 2
```

Comment:
This mission should have a blocked dialog in case the player doesn't have the necessary space.